bankroll/model/position.py

```diff
@@ -44,6 +45,13 @@ def __post_init__(self) -> None:
                 f"Cost basis {self.costBasis!r} should be zero if quantity is zero"
             )
 
+    @staticmethod
+    def dataframeColumns() -> List[str]:
+        return ["Instrument", "Quantity", "Avg Price"]
```

Comment:
Ideally, I'd use a tuple to relate the function call to obtain the value to the header column it should go under as a private mapping. This would allow the order of headers/values to be group correctly.

Reply:
What didn't work when you tried that?

Reply:
I wasn't sure how it would work with all of the following functions to obtain values:
```
        return [
            self.date.date(),
            self.action,
            abs(self.quantity),
            self.instrument,
            self.amount,
            self.fees,
        ]
```
…specifically `abs(self.quantity)`

Reply:
They're all of kind `Position -> Any`, so the functions have uniform type.

Reply:
✅ Got this sorted out nicely